js-api-spec/compiler.test.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright 2023 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {initCompiler, initAsyncCompiler} from 'sass';
+import type {Compiler, AsyncCompiler} from 'sass';
+
+import {sandbox} from './sandbox';
+
+describe('Compiler', () => {
+  let compiler: Compiler;
+
+  beforeEach(() => {
+    compiler = initCompiler();
+  });
+
+  afterEach(() => {
+    compiler.dispose();
+  });
+
+  describe('compileString', () => {
+    it('performs multiple compilations', () => {
+      expect(compiler.compileString('$a: b; c {d: $a}').css).toBe(
+        'c {\n  d: b;\n}'
```

Comment:
You can also use `toEqualIgnoringWhitespace` so you don't need to worry about the line breaks, for instance- `toEqualIgnoringWhitespace('c {d: b;}')`

Reply:
Thanks! Fixed in https://github.com/sass/sass-spec/pull/1954/commits/89ef7e5775b6a89e13e36c03f253e3bef169ccc8